requirements/deploy.txt

```diff
@@ -258,15 +258,15 @@ markdown==3.6
     # via -r requirements/pip.txt
 matplotlib-inline==0.1.7
     # via ipython
-newrelic==9.10.0
+newrelic==9.11.0
```

Comment:
This version supports pysopg3+

> Add support for [psycopg 3.0+](https://pypi.org/project/psycopg/)
>    New instrumentation for psycopg 3.0+ has been added, providing database tracing for both the Connection and AsyncConnection classes.

so, we should be able to upgrade our dependency as well https://github.com/readthedocs/readthedocs.org/issues/10596